### PR TITLE
one of the FF dipoles had the wrong sign defined - fixed

### PR DESCRIPTION
**Pull request by ajentsch**
### Briefly, what does this PR introduce?

Fixes the sign on a FF dipole which was incorrectly set.

### What kind of change does this PR introduce?
- [ x] Bug fix (issue #__)
- [ ] New feature (issue #__)
- [ ] Documentation update
- [ ] Other: __

### Please check if this PR fulfills the following:
- [ ] Tests for the changes have been added
- [ ] Documentation has been added / updated
- [ ] Changes have been communicated to collaborators

### Does this PR introduce breaking changes? What changes might users need to make to their code?

No, it corrects the transport for 100 GeV FF protons.

### Does this PR change default behavior?

Yes.

### CODE REVIEW

**Comment by simonge** [comment]
Presumably should be changed in the 5x100 configuration file too?

**Comment by ajentsch** [comment]
> Presumably should be changed in the 5x100 configuration file too?

Yes, good catch. New commit pushed.

File: compact/fields/beamline_10x100.xml [hunk 1]
```diff
@@ -44,7 +44,7 @@
     <constant name="Q1EF_GradientMax"  value="2.127596364*tesla/m"/>
 
     <constant name="B0PF_Bmax"  value="1.1840539*tesla"/>
-    <constant name="B0APF_Bmax" value="1.1588921*tesla"/> <!-- need to check with Scott on this one -->
+    <constant name="B0APF_Bmax" value="-1.1588921*tesla"/> <!-- need to check with Scott on this one -->
```
Comment (wdconinc) [inline compact/fields/beamline_10x100.xml:47]: Did you check with Scott yet? ;-) If so:
```suggestion
    <constant name="B0APF_Bmax" value="-1.1588921*tesla"/>
```

File: compact/fields/beamline_5x100.xml [hunk 2]
```diff
@@ -44,7 +44,7 @@
     <constant name="Q1EF_GradientMax"  value="2.127596364*tesla/m"/>
 
     <constant name="B0PF_Bmax"  value="1.1840539*tesla"/>
-    <constant name="B0APF_Bmax" value="1.1588921*tesla"/> <!-- need to check with Scott on this one -->
+    <constant name="B0APF_Bmax" value="-1.1588921*tesla"/> <!-- need to check with Scott on this one -->
```
Comment (wdconinc) [inline compact/fields/beamline_5x100.xml:47]:
```suggestion
    <constant name="B0APF_Bmax" value="-1.1588921*tesla"/>
```

Comment (ajentsch) [reply compact/fields/beamline_5x100.xml]: Done.

File: compact/fields/beamline_10x100.xml (hunk 1 above)
Comment (ajentsch) [reply compact/fields/beamline_10x100.xml]: Done.

**Review by wdconinc — APPROVED** [review]

**Pull request merged**